### PR TITLE
feat: make invite link more visible

### PR DESCRIPTION
**Pull request by sjaanus**
1. This is shortcut to invite more people to unleash
2. This is only visible to admins
3. Reusing the existing component

image
image

### CODE REVIEW

File: frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        <>
+            <Box
+                sx={{
+                    mb: {
+                        xs: 1,
+                        md: 0,
+                    },
+                    display: 'flex',
+                    justifyContent: 'center',
+                    flexDirection: 'column',
```
Comment (FredrikOseberg) [inline frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx:71]: could be styled component

File: frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            </Box>
+            <Box
+                sx={{
+                    display: 'flex',
+                    justifyContent: {
+                        xs: 'center',
+                        md: 'flex-end',
+                    },
+                    alignItems: 'center',
+                    flexGrow: 1,
```
Comment (FredrikOseberg) [inline frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx:120]: could be styled component

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Overall this looks nice, added a few minor comments.

However, on a medium sized window the popover will overflow the X axis and push the content out of bounds:

Skjermbilde 2023-10-11 kl  12 48 26

File: frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx (hunk 1 above)
Comment (sjaanus) [reply frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx]: Fixed

File: frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx (hunk 2 above)
Comment (sjaanus) [reply frontend/src/component/admin/users/InviteLinkBar/InviteLinkBarContent.tsx]: Fixed

**Comment by sjaanus** [comment]
> Overall this looks nice, added a few minor comments.
> 
> However, on a medium sized window the popover will overflow the X axis and push the content out of bounds:
> 
> Skjermbilde 2023-10-11 kl 12 48 26

Now it cant be pushed out of bounds.

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**